### ISSUE TITLE
TX-NR646: invalid path when creating a shortcut for TuneIn

### ISSUE THREAD

**Issue by mkulesh** [mkulesh/onpc#210]
When creating a shortcut for a station from TuneIn "My Presets" on TX-NR646,
additional "TuneIn Radio" is sometime added in front of the path:
```
CfgFavoriteShortcuts: Add favorite shortcut: 
<favoriteShortcut id="1" input="2B" service="0E" item="181.FM Kickin' Country (Country Music)" alias="181.FM Kickin' Country (Country Music)">
   <dir name="TuneIn Radio"/>
   <dir name="My Presets"/>
</favoriteShortcut>
```

The created shortcut is therefore invalid:
```
<onpcScript host="" port="" zone="0">
<send cmd="PWR" par="QSTN" wait="PWR"/>
<send cmd="PWR" par="01" wait="PWR" resp="01"/>
<send cmd="SLI" par="QSTN" wait="SLI"/>
<send cmd="SLI" par="2B" wait="SLI" resp="2B"/>
<send cmd="NLT" par="QSTN" wait="NLT"/>
<send cmd="NTC" par="TOP" wait="NLS" listitem="TuneIn Radio"/>
<send cmd="NSV" par="0E0" wait="NLA" listitem="TuneIn Radio"/>
<send cmd="NLA" par="TuneIn Radio" wait="NLA" listitem="My Presets"/>
<send cmd="NLA" par="My Presets" wait="NLA" listitem="181.FM Kickin' Country (Country Music)"/>
<send cmd="NLA" par="181.FM Kickin' Country (Country Music)" wait="1000"/>
</onpcScript>
```
![Screenshot_20210127-124418](https://user-images.githubusercontent.com/30557348/106158551-25a3c800-6184-11eb-9fe1-93e687ff703a.jpg)

**Comment by mkulesh** [comment]
Released in version 2.9.5 (iOS) and 2.9.4 (Android)

**Comment by mkulesh** [comment]
To be fixed in 1.22

**Comment by mkulesh** [comment]
Released in version 1.22
